src/components/Tool/FixSendNewsletter/Contact.tsx

```diff
@@ -98,18 +99,36 @@ const Contact = ({
   setContactFocus,
 }: Props): ReactElement => {
   const { t } = useTranslation();
-  const [newsletter, setNewsletter] = useState('BOTH');
+  const [newsletter, setNewsletter] = useState(SendNewsletterEnum.Both);
```

Comment:
As you set `newsletter` to `SendNewsletterEnum.None` in the useEffect, is there a reason you set it to `Both` on the initial render?
Could you set it as `None` here aswell?